spec/memo_wise_spec.rb

```diff
@@ -188,6 +188,21 @@ def public_memowise_method
       expect(instance.with_positional_args_counter).to eq(4)
     end
 
+    it "resets memoization for methods for specific positional arguments" do
+      instance.with_positional_args(1, 2)
+      instance.with_positional_args(2, 3)
+      instance.reset_memo_wise(:with_positional_args, 1, 2)
+
+      expect(Array.new(4) { instance.with_positional_args(1, 2) }).
+        to all eq("with_positional_args: a=1, b=2")
+
+      expect(Array.new(4) { instance.with_positional_args(2, 3) }).
+        to all eq("with_positional_args: a=2, b=3")
+
+      # This should be executed twice for each set of arguments passed
```

Comment:
Shouldn't this say ", and a third time for the set of arguments that was reset" or similar?